example_configs/cassandra.yml

```diff
@@ -2,12 +2,19 @@
 lowercaseOutputLabelNames: true
 lowercaseOutputName: true
 rules:
-- pattern: org.apache.cassandra.metrics<type=(Connection|Streaming), scope=(\S*), name=(\S*)><>(Count|Value)
+- pattern: org.apache.cassandra.metrics<type=(HintedHandOffManager)(?:, ((?!scope)\S*)=(\S*))?(?:, scope=(\S*))?,
```

Comment:
Can you give an example of the before and after?

Reply:
See commit message for a092b3e517f3d655a3e0e07340b65bbf3f2c55b9.

Reply:
This config doesn't seem to match the example output. Are you sure this is what you're using?

If you could share the help string that'd help my understanding.